receiver/statsdreceiver/README.md

```diff
@@ -43,13 +62,12 @@ General format is:
 
 `<name>:<value>|c|@<sample-rate>|#<tag1-key>:<tag1-value>`
 
+it supports sample rate
+
 ### Gauge
 
 `<name>:<value>|g|@<sample-rate>|#<tag1-key>:<tag1-value>`
 
-### Timer
```

Comment:
I'm curious why you're removing this?  We're going to need support for "ms" and "h" types eventually, should we instead have a TODO about adding support?

Reply:
Make sense, will revise